xbmc/cores/RetroPlayer/cheevos/Cheevos.cpp

```diff
@@ -108,6 +122,22 @@ bool CCheevos::LoadData()
   m_richPresenceScript = data[PATCH_DATA][RICH_PRESENCE].asString();
   m_richPresenceLoaded = true;
 
+  const CVariant& achievements = data[PATCH_DATA][ACHIEVEMENTS];
+  for (auto it = achievements.begin_array(); it != achievements.end_array(); it++)
+  {
+    const CVariant& achievement = *it;
+    if (achievement[FLAGS].asUnsignedInteger() == 3)
```

Comment:
what does `3` represent?

Reply:
Hi @Irusak, here 3 is just an id for official achievements in retroachievements org.

Reply:
"A magic number is a unique value with unexplained meaning which could (preferably) be replaced with a named constant."

I think @lrusak is making the point that `3` here is a magic number and should be a constant at the top of the file.